app/views/admin/friends/index.html.erb

```diff
@@ -19,12 +19,15 @@
         <% end %>
         <th>Phone Number</th>
         <th>Created</th>
+        <th>Entered</th>
```

Comment:
Because we have limited space in the table for columns, I don't think we can add this column.  I asked Sara when we were going over new features if there were any other columns that were needed in this table and they opted against adding any new ones.  They try to get 3 phone numbers for each friend and also add notes for each number (relationship of the person to the friend, etc.)  As a result, the phone number column takes up more space in production, but having all phone numbers for a friend in the table is a very high priority for them.  Hopefully, using the 1-year-deadline filter date range in combination with the ordering will be able to provide the same information.

Reply:
Okay, removing!